Classes/Emogrifier.php

```diff
@@ -122,6 +122,12 @@ class Emogrifier
     private $isStyleBlocksParsingEnabled = true;
 
     /**
+     * Determines whether elements with the `display: none` property are
+     * removed from the DOM.
+     */
+    private $isRemoveDisplayNoneEnabled = true;
```

Comment:
Actually, the option name is a bit misleading. As it is about removing/keeping nodes that are display: none (and not about removing/keeping the display: none itself), I propose the name "shouldKeepInvisibleNodes". What do you think?